src/meltano/core/tracking/schemas/README.md

```diff
@@ -0,0 +1,14 @@
+When a JSON file in this directory is updated, it must also be updated in our schema registry,
+SnowcatCloud.
+
+Login to https://app.snowcatcloud.com/ using your Meltano email address. Request permissions from
+an account admin if necessary. As of the time of writing this, the admins are:
+
+- pat@meltano.com
+- will@meltano.com
+- edgar@meltano.com
+- taylor@meltano.com
+- florian@meltano.com
```

Comment:
It's satisfying how each name (listed in the order we signed up in) is one character longer than the last.

Reply:
We just need to add 'aj@' to the top of the list 😆 

Except I'm not actually an admin.

🤔 ...if only I knew who to ask... 🤔 

Okay, jk. I couldn't resist.